plugins/generator-1.20.1/forge-1.20.1/templates/overlay.java.ftl

```diff
@@ -91,16 +90,32 @@ package ${package}.client.screens;
             </#list>
 
             <#list data.getComponentsOfType("Label") as component>
-                <#assign x = component.x - 213>
-                <#assign y = component.y - 120>
                     <#if hasProcedure(component.displayCondition)>
                         if (<@procedureOBJToConditionCode component.displayCondition/>)
                     </#if>
-                    event.getGuiGraphics().drawString(Minecraft.getInstance().font,
-                        <#if hasProcedure(component.text)><@procedureOBJToStringCode component.text/><#else>Component.translatable("gui.${modid}.${registryname}.${component.getName()}")</#if>,
-                        posX + ${x}, posY + ${y}, ${component.color.getRGB()}, false);
+                    <#if component.getAnchorPoint().name() == "TOP_LEFT">
+                    	posX = ${component.x};
+                    	posY = ${component.y};
+					<#elseif component.getAnchorPoint().name() == "TOP_RIGHT">

```

Comment:
The indentation here seems wrong. Check the rest of this portion of code as it's not the only line to have problems.

Reply:
Kinda LGTM (another request is already published)